### PR TITLE
DellEMC Z9332f: Fix SFP issue

### PR DESCRIPTION
**Pull request by aravindmani-1**
#### Why I did it
Some of the sonic-mgmt test cases failed in DellEMC Z9332f switch 
#### How I did it
Handled the platform API to fix the issue  
#### How to verify it
Run sonic-mgmt SFP test cases.
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205

#### Description for the changelog
UT logs:
[SFP UT.txt](https://github.com/sonic-net/sonic-buildimage/files/9401012/SFP.UT.txt)

### CODE REVIEW

File: platform/broadcom/sonic-platform-modules-dell/z9332f/sonic_platform/sfp.py [hunk 1]
```diff
@@ -1293,17 +1323,42 @@ def get_intl_state(self):
...
+            for i in range(4):
+                mask = (1 << i)
+                if not (channel & mask):
+                    continue
+                if disable:
+                    channel_state |= mask
+                else:
+                    channel_state &= ~mask
+            return self._write_eeprom_bytes(86, 1, bytearray([channel_state]))
+        else:
```
Comment (prgeor) [inline platform/broadcom/sonic-platform-modules-dell/z9332f/sonic_platform/sfp.py:1350]: does this work for SFP?

**Comment by prgeor** [comment]
> /easycla

@aravindmani-1 you need to sign the EasyCLA. Click on the "Details" and proceed

**Review by prgeor — APPROVED** [review]

File: platform/broadcom/sonic-platform-modules-dell/z9332f/sonic_platform/sfp.py (hunk 1 above)
Comment (aravindmani-1) [reply platform/broadcom/sonic-platform-modules-dell/z9332f/sonic_platform/sfp.py:1355]: No

**Pull request merged**